php/ext/google/protobuf/message.c

```diff
@@ -51,28 +51,21 @@
 // -----------------------------------------------------------------------------
 
 typedef struct {
-  zend_object std;
   zval arena;
   const Descriptor* desc;
   upb_Message *msg;
+  zend_object std;
```

Comment:
please add a comment explaining that this must come last because PHP uses a tail allocation to do more fun stuff